File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/non-plagiarized/05/T05.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function reverse(number: number): void {
  let balik = '';
  while (number > 0) {
    balik += (number % 10).toString();
    number = Math.floor(number / 10);
  }
  console.log(balik);
}

rl.question('Enter an integer: ', (input) => {
  const number = parseInt(input);
  reverse(number);
  rl.close();
});

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/non-plagiarized/05/T05.py
import sys

def reverse(number):
    balik = ""
    while number > 0:
        balik += str(number % 10)
        number //= 10
    print(balik)

if __name__ == "__main__":
    number = int(input("Enter an integer: "))
    reverse(number)

